### ISSUE TITLE
support gradient accumulation

### ISSUE THREAD

**Issue by wangraying** [BaguaSys/bagua#369]
refer to: https://pytorch.org/docs/stable/_modules/torch/nn/parallel/distributed.html#DistributedDataParallel.no_sync

**Comment by shjwudp** [comment]
it is now supported by https://github.com/BaguaSys/bagua/pull/312
